Mathlib/LinearAlgebra/Matrix/Trace.lean

```diff
@@ -4,6 +4,7 @@ Released under Apache 2.0 license as described in the file LICENSE.
 Authors: Johannes Hölzl, Patrick Massot, Casper Putz, Anne Baanen
 -/
 import Mathlib.Data.Matrix.Basic
+import Mathlib.Data.Matrix.Block
```

Comment:
I note that the only file imported by `Mathlib.Data.Matrix.Block` is `Mathlib.Data.Matrix.Basic` so there is no transitive impact of adding this import here.